### ISSUE TITLE
1.11.0 Release Tracking

### ISSUE THREAD

**Issue by chrisohaver** [miekg/coredns#6102]
This issue is for tracking the 1.11.0 release.

I think we have some pending changes that may cause backward incompatibles, hence the minor version bump.

The binary build step for the release will need to be done manually, since the process is no longer fully automated. Expect bumps. Buckle up.

**Comment by chrisohaver** [comment]
Let's see if someone has time to review #6120 so we can get that included in the release.

Any other open PRs that we should hold the release for?

**Comment by chrisohaver** [comment]
> Any other open PRs that we should hold the release for?

No, looks like all others are incomplete: (WIP or need update from PR author).

**Comment by chrisohaver** [comment]
Any open bugs of concern?  e.g. possible regressions?

#6122 suggests we should revert miekg/dns, but I disagree. I don't think we need to revert.
#6111 possible breakage of cache prefetch?  should we take a look at this before releasing?
#6087 seems like it would be an easy fix

**Comment by Tantalor93** [comment]
#6122 I think revert is not needed
#6111 imho can be fixed as part of patch release
#6087 looks like a good candidate to fix before doing this release
#6120 would be nice to include in this minor release as there is change in DoHWriter which might affect users of DohWriter and might force them to make slight code adjustments (stop passing Writer into it)

**Comment by chrisohaver** [comment]
I cant build CoreDNS at the moment.  I'm getting a slew of odd Go module errors, including:

```
go: cloud.google.com/go/compute@v1.19.0: Get "https://proxy.golang.org/cloud.google.com/go/compute/@v/v1.19.0.mod": write tcp [2600:4040:2ff5:9400:9cd9:4393:c94d:a7bb]:64588->[2607:f8b0:4004:c06::8d]:443: write: socket is not connected
```

**Comment by chrisohaver** [comment]
It was my employer's VPN interfering. All good now.

**Comment by chrisohaver** [comment]
With #5944 reverted (for now), There are not any backward incompatible changes that I'm aware of, and we can drop this back to a 1.10.2 release.

So we could release as is now on 1.10.2, or wait for #5944 to be resubmitted with fixes, and review/merge #6120 and release 1.11.0.

**Comment by chrisohaver** [comment]
Need to get #6151 sorted out before releasing.

**Comment by slonka** [comment]
Hi all :) could we get a patch release that updates `golang.org/x/net v0.4.0` to something that is not [vulnerable](https://dso.docker.com/cve/CVE-2022-41723)?

**Comment by chrisohaver** [comment]
Maintainers, are we comfortable releasing without https://github.com/coredns/coredns/issues/6151 resolved?  The root cause is still unclear/unknown, and the issue is not easily reproducible.

**Comment by chrisohaver** [comment]
I merged the 1.11.0 draft notes. They are not yet updated with recent merges or a highlights paragraph.  i.e. they should be updated before we release.

**Comment by yongtang** [comment]
Release draft is available: https://github.com/coredns/coredns/releases/tag/untagged-901b9b1b04eaf6b4a890

**Comment by yongtang** [comment]
Released 1.11.0: https://github.com/coredns/coredns/releases/tag/v1.11.0
